update-dependencies/CliDependencyHelper.cs

```diff
@@ -39,7 +39,7 @@ private static string GetCommitHash(string cliVersion)
         {
             using (ZipArchive archive = DownloadCliInstaller(cliVersion).Result)
             {
-                ZipArchiveEntry versionTxtEntry = archive.GetEntry($"sdk/{cliVersion}/.version");
+                ZipArchiveEntry versionTxtEntry = archive.GetEntry($"sdk\\{cliVersion}\\.version");
```

Comment:
I looked briefly and didn't see a change that looked like it caused this.  The change makes sense however since this is a windows zip.